include/cantera/oneD/StFlow.h

```diff
@@ -320,6 +319,7 @@ class StFlow : public Domain1D
             m_rho[j] = m_thermo->density();
             m_wtm[j] = m_thermo->meanMolecularWeight();
             m_cp[j] = m_thermo->cp_mass();
+            m_thermo->getPartialMolarEnthalpies(&m_hk(0,j));
```

Comment:
```suggestion
            m_thermo->getPartialMolarEnthalpies(&m_hk(0, j));
```
whitespace after comma.

Reply:
Corrected.